fastchat/serve/openai_api_server.py

```diff
@@ -87,6 +87,41 @@ async def check_model(request) -> Optional[JSONResponse]:
             )
     return ret
 
+async def check_length(request, prompt):
+    async with httpx.AsyncClient() as client:
+        worker_addr = await _get_worker_address(request.model, client)
+
+        response = await client.post(
+            worker_addr + "/model_details",
+            headers=headers,
+            json={},
+            timeout=WORKER_API_TIMEOUT,
+        )
+        context_len = response.json()["context_length"]
+
+        response = await client.post(
+            worker_addr + "/count_token",
+            headers=headers,
+            json={"prompt": prompt},
+            timeout=WORKER_API_TIMEOUT,
+        )
+        token_num = response.json()["count"]
+    
+    max_new_tokens = request.max_tokens
+    context_len = 2048
```

Comment:
add TODO? different models have different context length

Reply:
I made a mistake here, `the context_len` is already taken from the above request, this line should be deleted.